.github/workflows/submodule_update.yml

```diff
@@ -4,50 +4,71 @@ on:
   push:
     branches:
       - 'master'
-      - '2.10'
+      - '[2-9].\d+'
 
 jobs:
   tarantool-ee:
     # Run on push to the 'master' and release branches in tarantool/tarantool
     # repo only.
     if: github.repository == 'tarantool/tarantool'
 
-    runs-on: ubuntu-20.04-self-hosted
+    runs-on: [self-hosted, Linux, x86_64, flavor-1-2]
 
     env:
-      REPOSITORY: 'tarantool/tarantool-ee'
-      CHECKOUT_BRANCH: ${{ github.ref_name }}
-      FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
-      SUBMODULE: 'tarantool'
-      UPDATE_TO: ${{ github.ref_name }}
-      PR_AGAINST_BRANCH: ${{ github.ref_name }}
+      EE_FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
+      SDK_FEATURE_BRANCH: 'TarantoolBot/test-ce-${{ github.ref_name }}'
```

Comment:
Shouldn't it be `TarantoolBot/test-tarantool-${{ github.ref_name }}`?

Reply:
I want to use a single pattern for all integration test branches:
```
# submodule update that can be merged
TarantoolBot/update-tarantool-<version>
# submodule update originating from EE, not to be merged, just testing
TarantoolBot/test-ee-<feature-branch-name>
# submodule update originating from CE, going to EE and then SDK,
# also not to be merged, just testing
TarantoolBot/test-ce-<feature-branch-name>
```

Reply:
Got it, thanks.